docs/faq_debug.md

```diff
@@ -42,10 +42,10 @@ Sometimes it's useful to print debug messages from within your [custom code](cus
 
 After that you can use a few different print functions:
 
-* `print("string")`: Print a simple string.
-* `uprintf("%s string", var)`: Print a formatted string
-* `dprint("string")` Print a simple string, but only when debug mode is enabled
-* `dprintf("%s string", var)`: Print a formatted string, but only when debug mode is enabled
+* `print("string\n")`: Print a simple string.
+* `uprintf("%s string\n", var)`: Print a formatted string
+* `dprint("string\n")` Print a simple string, but only when debug mode is enabled
+* `dprintf("%s string\n", var)`: Print a formatted string, but only when debug mode is enabled
```

Comment:
Honestly, I think it might be a good idea to add a note here about the newline issue.